### ISSUE TITLE
xrandr block with gdm3

### ISSUE THREAD

**Issue by euri10** [greshake/i3status-rust#1697]
there are  long standing issue in gdm3 that spams syslog everytime xrandr asks for the monitors sizes
(https://gitlab.gnome.org/GNOME/gdm/-/issues/805 and https://gitlab.gnome.org/GNOME/gdm/-/issues/612
)

so I discovered I face the same because the xrandr block triggers periodically a `xrandr --listactivemonitors` 

I know this is  gdm3 issue, but if there was a way to work around it would be great, 

If not I would gladly add some sort of warning in the docs: having a syslog full of crap is not fun

**Comment by MaxVerevkin** [comment]
The docs already mention #274 and #668 and advise to use a higher update interval. Is your issue related? In any case, improving the docs is always welcomed.

> but if there was a way to work around it would be great,

Some discussion here: #1433

**Comment by euri10** [comment]
yes it's exactly what's reported in the 2 issues you linked that I'm facing,

one thing that would be interesting would be to check if affected users effectively use gdm3 which seems to me the root of the spam, not xrandr, and those 2 reports never mention it.

I can for myself say that I see only this on my laptop where I use gdm3 and not on my desktop where I'm using lightdm
